modules/installation/pages/general-requirements.adoc

```diff
@@ -42,21 +42,24 @@ Create an account with {scc} before installation of {sles} and {productname}.
 
 Depending on your organization's setup, you might also need to activate your subscription, using the btn:[Activate Subscriptions] menu.
 
+For more information about using SCC, go to https://scc.suse.com/docs/help.
 
 
 [[install.media]]
-== Obtain the Unified Installer
+== Unified Installer
 
-{susemgr} Server and Proxy can be installed with the {sle} Unified Installer.
+{susemgr} Server and Proxy are installed with the {sle} Unified Installer.
 
 //REMARK What about Uyuni?
 ifeval::[{suma-content} == true]
 You only require a valid registration code for {productname}.
 You do not require a separate code for SLES{nbsp}{sles-version} {sp-version}.
 endif::[]
 
-If not already done, download the {sle} Unified Installer from https://download.suse.com/index.jsp.
-Direct link to {sle} {sles-version} {sp-version}, required to install SUSE Manager 4.1: https://download.suse.com/index.jsp?product_id=&search=Search&families=22609&version=68287.
+If not already done, download the {sle} Unified Installer from https://download.suse.com.
```

Comment:
Did we already explain somewhere what the Unified Installer is?  Two sentences are probably sufficient: With the Unified Installer you can install all products that are part of SLE. The Unified Installer is SLE installation media.

Reply:
@keichwa - Sort of. In the last sentence of this very chapter UI is described. From my pow this is not the most elegant place for definition, we should probably start the chapter with it with it, but that would require rewriting of the current first sentence.